shared/src/persistence/dynamo/users/createPractitionerUser.js

```diff
@@ -52,10 +52,15 @@ exports.createUserRecords = async ({ applicationContext, user, userId }) => {
 
 exports.createPractitionerUser = async ({ applicationContext, user }) => {
   let userId = applicationContext.getUniqueId();
+  const practitionerRoleTypes = [
+    ROLES.privatePractitioner,
+    ROLES.irsPractitioner,
+    ROLES.inactivePractitioner,
+  ];
 
-  if (![ROLES.privatePractitioner, ROLES.irsPractitioner].includes(user.role)) {
+  if (!practitionerRoleTypes.includes(user.role)) {
     throw new Error(
-      'Practitioner users must have either private or IRS practitioner role',
+      'Practitioner users must have either private IRS practitioner role, or be an inactive practitioner',
```

Comment:
This message reads a bit clunky right now - maybe instead say something like, `Practitioner users must have private, IRS, or inactive practitioner roles`.

Reply:
Agreed, update coming!

Reply:
I commented similarly in the original PR earlier this morning; just removed my comment. Thanks for the catch @julialeague !

Reply:
Updated!